libraries/AP_Camera/AP_Camera_MAVLinkCamV2.cpp

```diff
@@ -60,7 +64,7 @@ bool AP_Camera_MAVLinkCamV2::record_video(bool start_recording)
     }
 
     _link->send_message(MAVLINK_MSG_ID_COMMAND_LONG, (const char*)&pkt);
-
+    is_recording = start_recording;
```

Comment:
I don't think we need this because we use feedback from the camera to decide if recording has actually started or not.